src/test/java/org/minions/devfund/alejandro/movie/database/ActorTests.java

```diff
@@ -0,0 +1,43 @@
+package org.minions.devfund.alejandro.movie.database;
+
+import org.junit.Test;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * This class contains all test for Actor class.
+ */
+public class ActorTests {
+
+    /**
+     * This method verify the Actor constructor.
+     */
+    @Test
+    public void testActorConstructor() {
+        final String actorName = "test123";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "test123" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)